<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileReplaceDialog</class>
 <widget class="QDialog" name="FileReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="srcLabel">
     <property name="text">
      <string>src</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&gt;&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dstLabel">
     <property name="text">
      <string>dst</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>8</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="applyAllCheckBox">
       <property name="text">
        <string>Apply to all</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yesButton">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noButton">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>yesButton</tabstop>
  <tabstop>noButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>applyAllCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
